### ISSUE TITLE
Error when deploying nanodet

### ISSUE THREAD

**Issue by CheungBH** [cmdbug/TNN_Demo#15]
Hello. I trained a nanodet model from 
https://github.com/RangiLyu/nanodet, 
and an error occurred to me when processing
    _auto status = NanoDet::instance->GetCommandQueue(&command_queue);_
:

2021-10-03 22:27:42.800142+0800 TNNDemo[7059:1059735] new NanoDet
E/tnn: virtual tnn::Status tnn::DefaultNetwork::InitLayers(tnn::NetStructure *, tnn::NetResource *) [File /Users/TURUI/Desktop/TNN/source/tnn/core/default_network.cc][Line 236] Error Init layer Conv_0 (err: 4096 or 0x1000)
2021-10-03 22:27:42.839470+0800 TNNDemo[7059:1059735] GPU initialization failed, switch to CPU
E/tnn: virtual tnn::Status tnn::DefaultNetwork::InitLayers(tnn::NetStructure *, tnn::NetResource *) [File /Users/TURUI/Desktop/TNN/source/tnn/core/default_network.cc][Line 236] Error Init layer Conv_0 (err: 4096 or 0x1000)
2021-10-03 22:27:42.845144+0800 TNNDemo[7059:1059735] TNN init failed 4096
(lldb) 

It seems that the error is caused by initialization. Do you have any ideas to fix it?

**Comment by cmdbug** [comment]
It seems that there is something wrong with the conv layer. Try using the latest code conversion and Library of TNN. Or have you made any other changes?

**Comment by CheungBH** [comment]
> It seems that there is something wrong with the conv layer. Try using the latest code conversion and Library of TNN. Or have you made any other changes?

No, I didn't change anything. And both your demo and TNN library are the latest. Could you pls check the model? I can upload the model.

**Comment by CheungBH** [comment]
https://drive.google.com/file/d/1cDdwT2sKFIbGsFU1DkuYDc5JiCFrdyWZ/view?usp=sharing

Here's the model. The onnx model outside is the source model, while the two folders contain the converted models and the models with "--half"

**Comment by CheungBH** [comment]
Hi. Do you have ideas to fix this bug? I tried using the newest code to convert the model, but it also fails.

**Comment by cmdbug** [comment]
See which layer conv0 is and whether the parameters are correct. Let's see under what circumstances this error will be reported in the initialization part of the TNN source code.
